var/spack/repos/builtin/packages/nalu/package.py

```diff
@@ -35,8 +35,6 @@ class Nalu(CMakePackage):
     homepage = "https://github.com/NaluCFD/Nalu"
```

Comment:
Might want to remove "Wind ECP" from the docstring now that it has its own package.